wellregistry/registry/tests/test_models.py

```diff
@@ -0,0 +1,211 @@
+
+from decimal import Decimal
```

Comment:
This is basically the same test code that was in test_admin to test the validations